cadc-registry/src/main/java/ca/nrc/cadc/reg/client/LocalAuthority.java

```diff
@@ -87,26 +88,19 @@
  * @author pdowler
  */
 public class LocalAuthority {
-
     private static final Logger log = Logger.getLogger(LocalAuthority.class);
 
-    private static final String LOCAL_AUTH_PROP_FILE = LocalAuthority.class.getSimpleName() + ".properties";
-
-    private Map<URI, Set<URI>> authorityMap = new TreeMap<>();
+    private static final Set<URI> EMPTY_SET = Collections.unmodifiableSet(Collections.EMPTY_SET);
+    
+    private final Map<URI, Set<URI>> authorityMap = new TreeMap<>();
 
     public LocalAuthority() {
         PropertiesReader propReader = new PropertiesReader(RegistryClient.CONFIG_FILE);
         MultiValuedProperties mvp = propReader.getAllProperties();
         
-        // backwards compat: try the old config file name
-        if (mvp.isEmpty()) {
-            propReader = new PropertiesReader(LOCAL_AUTH_PROP_FILE);
-            mvp = propReader.getAllProperties();
-        }
-
         for (String std : mvp.keySet()) {
             List<String> values = mvp.getProperty(std);
-            Set<URI> vals = new HashSet<>();
+            Set<URI> vals = new TreeSet<>(); // TreeSet is sorted so get-first will be predictable
```

Comment:
Get-first in alphabetical order? Isn't it more intuitive to have it in the order found in the config file?